internal/toolkit/toolkit_test.go

```diff
@@ -126,6 +126,17 @@ func TestToolkit(t *testing.T) {
 				Expect(ok).To(BeFalse())
 			})
 
+			it("gets input list", func() {
+				tk.Environment = map[string]string{"INPUT_BLOCKED_NAMESPACES_TEST-1": "test-value-1,test-value-2,test-value-3"}
+
+				s, ok := tk.GetInputList("test-1")
```

Comment:
I don't think this should pass. Shouldn't it be `tk.GetInputList("blocked-namespaces-test-1")`?

Reply:
I guess, it was passing, however, I will check again and revert?

Reply:
Yes, my bad, just fixed it, the tests are passing now